creating-using-web.md

```diff
@@ -25,6 +23,13 @@ learner shares their screen (video) and others can ask questions and give
 suggestions. In a larger group the instructor can demonstrate these and then
 participants can try on their own and then ask questions.
 
+
+### Note to instructors
```

Comment:
Doesn't need to be done now, but we can make a "note to instructors" directive to wrap this stuff and minimize the obtrusiveness via CSS.

Reply:
I agree but I wanted to discuss the CSS setup anyway soon separately.